pkg/stacks/templates/engines/util.go

```diff
@@ -0,0 +1,41 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package engines
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/kubectl/pkg/util/hash"
```

Comment:
i think this is the first and only usage of bringing in the `kubectl` package dependency.  Should we bringing in `kubectl` as a package just for this hashing funtionality? how important is it? would it be easier/wiser to copy it?

Consider https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s from https://go-proverbs.github.io/ :)

Reply:
Hmmm! Potentially then! I don't feel like a go expert yet, so I like these links o: . I may revisit this one in a future change.